content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -57,58 +60,65 @@ For example:
+   {
+     "kind": "PodList",
+     "apiVersion": "v1",
+     "metadata": {"resourceVersion":"10245"},
+     "items": [...]
+   }
+   ```
 
 2. Starting from resource version 10245, receive notifications of any creates, deletes, or updates as individual JSON objects.
 
-        GET /api/v1/namespaces/test/pods?watch=1&resourceVersion=10245
-        ---
-        200 OK
-        Transfer-Encoding: chunked
-        Content-Type: application/json
-        {
-          "type": "ADDED",
-          "object": {"kind": "Pod", "apiVersion": "v1", "metadata": {"resourceVersion": "10596", ...}, ...}
-        }
-        {
-          "type": "MODIFIED",
-          "object": {"kind": "Pod", "apiVersion": "v1", "metadata": {"resourceVersion": "11020", ...}, ...}
-        }
-        ...
+   ```
+   GET /api/v1/namespaces/test/pods?watch=1&resourceVersion=10245
+   ---
+   200 OK
+   Transfer-Encoding: chunked
+   Content-Type: application/json
+   {
+     "type": "ADDED",
+     "object": {"kind": "Pod", "apiVersion": "v1", "metadata": {"resourceVersion": "10596", ...}, ...}
+   }
+   {
+     "type": "MODIFIED",
+     "object": {"kind": "Pod", "apiVersion": "v1", "metadata": {"resourceVersion": "11020", ...}, ...}
+   }
+   ...
+   ```
 
 A given Kubernetes server will only preserve a historical list of changes for a limited time. Clusters using etcd3 preserve changes in the last 5 minutes by default.  When the requested watch operations fail because the historical version of that resource is not available, clients must handle the case by recognizing the status code `410 Gone`, clearing their local cache, performing a list operation, and starting the watch from the `resourceVersion` returned by that new list operation. Most client libraries offer some form of standard tool for this logic. (In Go this is called a `Reflector` and is located in the `k8s.io/client-go/cache` package.)
 
 ### Watch bookmarks
 
 To mitigate the impact of short history window, we introduced a concept of `bookmark` watch event. It is a special kind of event to mark that all changes up to a given `resourceVersion` the client is requesting have already been sent. Object returned in that event is of the type requested by the request, but only `resourceVersion` field is set, e.g.:
 
-        GET /api/v1/namespaces/test/pods?watch=1&resourceVersion=10245&allowWatchBookmarks=true
-        ---
-        200 OK
-        Transfer-Encoding: chunked
-        Content-Type: application/json
-        {
-          "type": "ADDED",
-          "object": {"kind": "Pod", "apiVersion": "v1", "metadata": {"resourceVersion": "10596", ...}, ...}
-        }
-        ...
-        {
-          "type": "BOOKMARK",
-          "object": {"kind": "Pod", "apiVersion": "v1", "metadata": {"resourceVersion": "12746"} }
-        }
+```console
+GET /api/v1/namespaces/test/pods?watch=1&resourceVersion=10245&allowWatchBookmarks=true
+---
+200 OK
+Transfer-Encoding: chunked
+Content-Type: application/json
```

Comment:
mini nit: HTTP/1.1 usually has a blank line between headers and body; I'd include that here (it looks odd to me without it)